quantum/rgb_matrix/rgb_matrix.c

```diff
@@ -164,6 +164,23 @@ void eeconfig_debug_rgb_matrix(void) {
     dprintf("rgb_matrix_config.flags = %d\n", rgb_matrix_config.flags);
 }
 
+uint32_t eeconfig_read_rgb_matrix(void) {
+#ifdef EEPROM_ENABLE
+    return eeprom_read_dword(EECONFIG_RGB_MATRIX);
+#else
+    return 0;
+#endif
+}
+
+void rgb_matrix_reload_from_eeprom(void) {
+    /* Reset back to what we have in eeprom */
+    rgb_matrix_config.raw = eeconfig_read_rgb_matrix();
```

Comment:
```suggestion
void rgb_matrix_reload_from_eeprom(void) {
    /* Reset back to what we have in eeprom */
    eeconfig_init_rgb_matrix();
```
`rgb_matrix_config.raw` does not cover the whole `rgb_config_t` (this member probably needs to be deprecated and eventually removed completely), but `EECONFIG_DEBOUNCE_HELPER()` already defines a suitable function to load the configuration from the EEPROM.